prereise/gather/griddata/hifld/const.py

```diff
@@ -113,8 +113,8 @@
     "eia_form860_2019_wind": "https://besciences.blob.core.windows.net/datasets/EIA_Form860/3_2_Wind_Y2019_Operable.csv",
     "epa_ampd": "https://besciences.blob.core.windows.net/datasets/EPA_AMPD/",
     "epa_needs": "https://besciences.blob.core.windows.net/datasets/EPA_NEEDS/needs-v620_06-30-21-2_active.csv",
-    "substations": "https://besciences.blob.core.windows.net/datasets/hifld/Electric_Substations_Jul2020.csv",
-    "transmission_lines": "https://besciences.blob.core.windows.net/datasets/hifld/Electric_Power_Transmission_Lines_Jul2020.geojson.zip",
+    "substations": "https://besciences.blob.core.windows.net/datasets/hifld/Electric_Substations_Apr2022.csv",
+    "transmission_lines": "https://besciences.blob.core.windows.net/datasets/hifld/Electric_Power_Transmission_Lines_Apr2022.geojson.zip",
     "us_counties": "https://besciences.blob.core.windows.net/datasets/geo_data/uscounties.csv",
     "us_zips": "https://besciences.blob.core.windows.net/datasets/geo_data/uszips.csv",
 }
```

Comment:
Should we define a `blob_storage_url` at the top of the file or in even better in a module located in PreREISE/prereise/utility?

Reply:
Done